ui-grid/ui-grid.d.ts

```diff
@@ -484,6 +484,10 @@ declare module uiGrid {
          */
         appScope?: ng.IScope;
         /**
+        * returns an array of columns in the grid
+        */
+        columns: Array<IGridColumn>;
```

Comment:
I tested this, and it does seem to work at runtime, but I can't find any reference to this in the documentation (http://ui-grid.info/docs/#/api/ui.grid.class:Grid).  Perhaps this would be a good opportunity to improve the docs?